### PR TITLE
Submit Update-2022-12.md

### PR DESCRIPTION
**Pull request by lukeschantz**
This is the December update for the Project Alpha jQuery grant.

Signed-off-by: lukeschantz <luke.schantz@gmail.com>

### CODE REVIEW

File: alpha/engagements/2022/jquery/Update-2022-12.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+##### 2.2.3) Security improvements
+
+The new servers have been implemented with several security measures, compared to the old droplets. Access logs are no longer kept on-server, traffic between CDN and origin server is now always encrypted, Linux upgraded from Debian 7 (end-of-life as of 2018) to Debian 11 LTS, and Node.js upgraded from v10 to v12. TLS 1.3 has also been enabled at the CDN edge, for improved security and performance. These updates ensure that the project's website remains secure and running efficiently.
+
+It was [identified](https://github.com/gruntjs/gruntjs.com/issues/221
+) that the gruntjs website relies on the presence of dev-dependencies on the server. It is recommended that the appropriate package be moved to the list of service dependencies instead, allowing the server to be provisioned using "npm --production" as is best practice to limit exposure of additional packages and their potential vulnerabilities and compromise.
```
Comment (JLLeitschuh) [inline alpha/engagements/2022/jquery/Update-2022-12.md:54]:
```suggestion
It was [identified](https://github.com/gruntjs/gruntjs.com/issues/221) that the gruntjs website relies on the presence of dev-dependencies on the server. It is recommended that the appropriate package be moved to the list of service dependencies instead, allowing the server to be provisioned using "npm --production" as is best practice to limit exposure of additional packages and their potential vulnerabilities and compromise.
```

**Review by lukeschantz — COMMENTED** [review]
Thank you for reviewing, correcting and approving this jQuery update.

**Comment by lukeschantz** [comment]
Is their anything else I need to do for this Update to be published?  Please advise.

**Comment by lukeschantz** [comment]
@JLLeitschuh Is there anything else you need or can this be merged? Thank you.

File: alpha/engagements/2022/jquery/README.md [hunk 2]
```diff
@@ -1 +1,30 @@
-# jQuery
+## Alpha Engagemnt: jQuery
```
Comment (bobcallaway) [inline alpha/engagements/2022/jquery/README.md:1]:
```suggestion
## Alpha Engagement: jQuery 
```

**Review by bobcallaway — APPROVED** [review]
lgtm other than one spelling nit

**Review by scovetta — APPROVED** [review]

**Pull request merged**